framework/src/Volo.Abp.Core/Volo/Abp/AbpApplicationCreationOptions.cs

```diff
@@ -19,10 +19,13 @@ public class AbpApplicationCreationOptions
     [NotNull]
     public AbpConfigurationBuilderOptions Configuration { get; }
 
+    public bool ManualConfigureServices { get; set; }
```

Comment:
What is that used for?

Reply:
The constructor of `AbpApplicationBase `will call the sync `ConfigureServices`, async initialization cannot be done in the constructor.

Reply:
![image](https://user-images.githubusercontent.com/6908465/147304892-7e250743-a5e6-4bfd-b9a1-22199b09d25b.png)